src/Frontend/App.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import Router from 'react-router';
+import routes from './config/routes';
+
+Router.run(routes, function (Root) {
+    React.render(<Root />, document.getElementById('content'));
```

Comment:
hmm, should we validate whether this element exist, so we should throw a proper error?

pseudocode:
``` javascript
let element = document.getElementById('content');
if (!element || element === undefined) {
    throw new Error('Cannot find page element');
}
```